### DISCUSSION TITLE
I'm blocked with "Error: invalid relative path: ../../usr/bin/ldd" when deploying my NextJS app with Vercel

### DISCUSSION THREAD

**Discussion by lauthieb** [vercel/community#6675] [Help]
### Summary

Hello, 

I published my portfolio for years built with NextJS and deployed on Vercel. But now, when I updated it, I now have this error:

`Error: invalid relative path: ../../usr/bin/ldd`

Can you please help me? Even if I deploy older source (that passes before), I still have this error :(

![CleanShot 2024-04-29 at 00 45 24@2x](https://github.com/vercel/next.js/assets/9600228/7d3051c5-11ad-4fc5-b425-63ba1949ee2d)

Thank you for your help.

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 23.4.0: Fri Mar 15 00:12:49 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T6020
    Binaries:
      Node: 20.10.0
      npm: 10.2.3
      Yarn: N/A
      pnpm: 9.0.6
    Relevant packages:
      next: 12.1.4
      react: 17.0.2
      react-dom: 17.0.2
```

### Steps to Reproduce

I don't know how to reproduce... Because my project always worked and now it is down. Unfortunately I cannot make the code open source :(

**Comment by sagish** [comment]
Try upgrading `next` to `12.3.4` or later
